### ISSUE TITLE
bundled builds don't add custom-elements-es5-adapter.js shim

### ISSUE THREAD

**Issue by atoy40** [Polymer/polymer-cli#710]
Hello,

Bundled builds don't add the custom-elements-es5-adapter.js shim and I think it's because addCustomElementsEs5Adapter() is added to the build pipeline after the bundler(), so the es5 adapter cannot find any webcomponent script tag it's searching for because it's already bundled into the html.

Thanks
Anthony.

**Comment by FredKSchott** [comment]
Hmm interesting. Do you have a `polymer.json` for your project or are you building without one?

**Comment by atoy40** [comment]
Yes, i've a polymer.json file. here it is ! :
```
{
  "entrypoint": "index.html",
  "sources": [
    "app/components/**/*"
  ],
  "extraDependencies": [
    "app/bower_components/webcomponentsjs/*.js"
  ],
  "builds": [
    {
      "name": "production",
      "bundle": true,
      "addServiceWorker": false,
      "js": {
          "minify": true,
          "compile": true
      },
      "css": {
          "minify": true
      },
      "html": {
          "minify": true
      }
    }
  ],
  "lint": {
    "rules": ["polymer-2"]
  }
}
```

**Comment by FredKSchott** [comment]
Alright that's very possible, let me try to reproduce to see how we can fix.

**Comment by atoy40** [comment]
May be just move the part of the code in build/build.ts that call addCustomElementsEs5Adapter (arround line 84) before the code that call bundler (arround 74) is enough  ? :)

**Comment by usergenic** [comment]
I feel like this was fixed.  Assigning to myself to verify.

**Comment by aarondrabeck** [comment]
This is still an issue in CLI v1.2.0

**Comment by usergenic** [comment]
Fixed in #817
